server/routes/index.ts

```diff
@@ -1,32 +1,30 @@
-import { type RequestHandler, Router } from 'express'
+/* istanbul ignore file */
 
-import { Controllers } from 'server/controllers'
-import asyncMiddleware from '../middleware/asyncMiddleware'
+import { Router } from 'express'
+import { Controllers } from '../controllers'
+import { Services } from '../services'
+import { actions } from './utils'
 import paths from '../paths/apply'
 import applyRoutes from './apply'
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-export default function routes(controllers: Controllers): Router {
+export default function routes(controllers: Controllers, services: Services): Router {
   const router = Router()
-  const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
-  const post = (path: string | string[], handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+  const { get, post } = actions(router, services.auditService)
 
-  get('/', (req, res, next) => {
-    res.render('pages/index')
-  })
+  get(
+    '/',
+    (req, res, next) => {
+      res.render('pages/index')
+    },
+    { auditEvent: 'VIEW_LANDING' },
+  )
 
-  const { applicationsController, peopleController } = controllers
+  const { peopleController } = controllers
 
-  get(paths.applications.new.pattern, applicationsController.new())
-
-  get(paths.applications.index.pattern, applicationsController.index())
-
-  get(paths.applications.show.pattern, applicationsController.show())
-
-  post(paths.applications.create.pattern, applicationsController.create())
   post(paths.applications.people.find.pattern, peopleController.find())
```

Comment:
do we need to do something here to explicitly audit "find by CRN" actions or is that handled magically? e.g.

```
post(paths.applications.people.find.pattern, peopleController.find(), { auditEvent: 'FIND_PERSON_BY_CRN' })
```

Reply:
oh good spot, we do need to manually add it I believe